content/webapp/pages/events/[eventId].tsx

```diff
@@ -505,7 +511,7 @@ const EventPage: NextPage<Props> = ({ event, jsonLd }) => {
   );
 };
 
-export const getServerSideProps: GetServerSideProps<Props> = async context => {
+const originalGSSP: GetServerSideProps<EventProps> = async context => {
```

Comment:
maybe call this getServerSidePropsForEvent

Reply:
this method is not going to be used for this